build/runtime.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+declare(strict_types=1);
+
+require 'phar://flow-php.phar/vendor/autoload.php';
+
+use Flow\ETL\DataFrame;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\SingleCommandApplication;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+(new SingleCommandApplication())
+    ->setName('Flow-PHP - Extract Transform Load - Data processing framework')
+    ->setVersion(FlowVersion::getVersion())
+    ->addArgument('input-file', InputArgument::REQUIRED, '')
+    ->setCode(function (InputInterface $input, OutputInterface $output): int {
+        $style = new SymfonyStyle($input, $output);
+
+        $filename = (string) $input->getArgument('input-file');
+        if (!\file_exists($filename)) {
+            $style->error("Input file ($filename) doesn't exists!");
+
+            return Command::FAILURE;
+        }
+
+        if (!\str_ends_with($filename, '.php')) {
```

Comment:
I think I would read the first 5 bytes of the file and check if it starts from `<?php` string (lower/upper case)